src/LiveComponent/doc/index.rst

```diff
@@ -384,6 +384,56 @@ This can be useful along with a button that triggers a render on click:
     <input data-model="norender|coupon">
     <button data-action="live#$render">Apply</button>
 
+Input Model Validation Modifiers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 2.28
+
+When binding data using the ``data-model`` directive, you can apply input validation modifiers such as:
+
+- ``min_length(<value>)``: Ensures the input length is at least the given number of characters.
+- ``max_length(<value>)``: Ensures the input length does not exceed the given number of characters.
+- ``min_value(<value>)``: Ensures the input value is not less than the given numeric value.
+- ``max_value(<value>)``: Ensures the input value is not greater than the given numeric value.
+
+These modifiers help reduce unnecessary server requests and provide a lightweight form of frontend validation.
+
+Example:
+
+.. code-block:: html
+
+    <input data-model="min_length(3)|username" type="text" value="" />
+    <input data-model="min_value(10)|max_value(100)|quantity" type="number" value="20" />
+
+In the example above:
+
+- The first input will not trigger a model update until at least 3 characters are entered.
+- The second input will only trigger updates when the number is between 10 and 100 (inclusive).
+
+Supported Input Types
+---------------------
+
+These modifiers are only evaluated on compatible input types:
+
+- ``min_length``, ``max_length``: Only valid for ``<input type="text">``, ``type="email"``, ``type="password"``, ``type="search"``, ``type="url"``, and ``<textarea>`` elements.
+- ``min_value``, ``max_value``: Only valid for ``<input type="number">``, ``type="range"``, and similar numeric fields.
+
+Combining with debounce
+-----------------------
+
+These validation modifiers can be combined with ``debounce``:
+
+.. code-block:: html
+
+    <input data-model="min_length(4)|debounce(300)|comment" type="text" />
+
+This prevents sending multiple Ajax requests for short inputs and ensures only meaningful data changes are transmitted after 300ms of inactivity. It's especially useful for reducing server load and avoiding over-rendering while typing.
```

Comment:
Maybe we can remove this section too, in favor of adding an example with `|debounce(x)` in the previous HTML examples